### PR TITLE
Fix Doc tests in Tox for torch

### PR DESCRIPTION
**Pull request by yeandy**
Add `__init__` to the `inference` folder, and fix linting errors.

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: sdks/python/scripts/generate_pydoc.sh [hunk 1]
```diff
@@ -235,17 +235,6 @@ nitpick_ignore = []
 nitpick_ignore += [('py:class', iden) for iden in ignore_identifiers]
 nitpick_ignore += [('py:obj', iden) for iden in ignore_identifiers]
 nitpick_ignore += [('py:exc', iden) for iden in ignore_references]
-
-# Monkey patch functools.wraps to retain original function argument signature
-# for documentation.
-# https://github.com/sphinx-doc/sphinx/issues/1711
```
Comment (yeandy) [inline sdks/python/scripts/generate_pydoc.sh:241]: This Sphinx [issue](https://github.com/sphinx-doc/sphinx/issues/1711) has been fixed, so removing this code

**Comment by AnandInguva** [comment]
Run PythonDocs PreCommit

**Comment by aaltay** [comment]
What is the next step on this PR?

**Comment by AnandInguva** [comment]
I think we created a new PR which reflects the changes in current PR.

**Comment by aaltay** [comment]
> I think we created a new PR which reflects the changes in current PR.

I am sorry I did not understand. Why do we need to create a new PR?

**Comment by AnandInguva** [comment]
Sorry. We merged another PR with these changes few days back. I think this can be closed now

**Pull request closed without merging**